tests/Form/Fields/PagesFieldTest.php

```diff
@@ -162,4 +162,29 @@ public function testRequiredValid()
 
         $this->assertTrue($field->isValid());
     }
+
+    public function testApiPicker()
+    {
+        $app = $this->app->clone([
+            'options' => [
+                'api.allowImpersonation' => true
+            ],
+            'blueprints' => [
+                'pages/default' => [
+                    'fields' => [
+                        'foo' => [
+                            'type' => 'pages',
+                        ]
+                    ]
+                ]
+            ]
+        ]);
+
+        $app->impersonate('kirby');
+        $response = $app->api()->call('pages/a/fields/foo');
+
+        $this->assertCount(2, $response['data']);
```

Comment:
@lukasbestle It passed when run one by one (files/pages/users field api tests), but fails when running all tests. Do you have an idea?

Reply:
I've fixed with separate model templates.